Home.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="Styles.css">
```

Comment:
Stuff you wrote yourself should be listed at the bottom of the head section. Stuff other people wrote should be at the top. This is best practise, so that you can quickly identify the code you need to review and the code written by others.